src/qibolab/instruments/abstract.py

```diff
@@ -73,15 +78,20 @@ def sampling_rate(self) -> int:
         (GSps)."""
 
     @abstractmethod
-    def play(self, *args, **kwargs):
+    def play(
+        self,
+        updates: list[ConfigUpdate],
```

Comment:
I encountered this issue yesterday while working #933. I think here we need
```suggestion
        config: dict[str, Config],
```
otherwise the instruments do not get any information about channels not used in the updates.

If you also see in `Platform.execute` a `config` dictionary is defined and updated (based on the given `updates`) in https://github.com/qiboteam/qibolab/blob/d9b72112d0f34a2855b34ab1184732e4e3051908/src/qibolab/platform/platform.py#L294-L295
but never used afterwards. It should be passed to `instrument.play`. This is fixed in #933 in order to test on hardware.

Reply:
I actually forgot the comment. However, since it is fixed in #933, I will try to address while rebasing on it